falkordb-node/node-entrypoint.sh

```
@@ -532,6 +532,8 @@ if [[ $DEBUG -eq 1 && $RUN_SENTINEL -eq 1 ]]; then
   done
 fi
 
+log $(getent hosts || cat /etc/hosts)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)